web/blog/vast-v3.0/index.md

```diff
@@ -0,0 +1,349 @@
+---
+draft: true
+title: VAST v3.0
+description: VASTQL Refreshments
+authors: [dominiklohmann, mavam]
+date: 2023-01-20
+image: /path/to/image.png
```

Comment:
Boilerplate comment to track making this image.

Reply:
I've put this as a TODO into the blog post itself; we want to merge this as a draft and then I will hand this over.